EdurecaInClassProject/src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="footer">
+
+    <div class="container">
+      <div class="row mt-5 mb-5">
+        <div class="col-md-6 ">
+          <div class="card card-body box">
+            <h5 class="text-center mb-3">Popular Roads</h5>
+          
+                <ul class="row">
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Bangalore to Hubli</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Mumbai to Pune</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Chennai to Hubli</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Chennai to Bangalore</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Pune to Bangalore</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Mumbai to Bangalore</li>
+                    
+                   
+                </ul>
+           
+          </div>
+        </div>
+  
+        <div class="col-md-6">
+        <div class="card card-body box" >
+            <div id="why_buy">                            
+                <div class="info_right">
+                    <h5 class="text-center mb-3">Why buy tickets from us?</h5>
+                    <ul>
+                        <li><i class="fas fa-check mr-2" style="color:#F69230;font-size:17px;"></i><b>Buy bus tickets</b> anytime from anywhere</li>
```

Comment:
Avoid style in HTML, use the CSS file.
It can also reduce the redundant code that is getting used multiple times.

Reply:
yeah sure